pkg/cluster/deploystorage.go

```diff
@@ -72,9 +72,15 @@ func (m *manager) ensureResourceGroup(ctx context.Context) error {
 		ManagedBy: to.StringPtr(m.doc.OpenShiftCluster.ID),
 	}
 	if m.env.IsLocalDevelopmentMode() {
-		// TODO: ideally this should be removed, but removing it causes the
-		// purge script to ignore RGs in CI E2E.  Fix that at the same time.
-		group.ManagedBy = nil
+		rg, err := m.resourceGroups.Get(ctx, resourceGroup)
+		if err == nil {
+			group.Tags = rg.Tags
+		}
```

Comment:
I would add a comment describing why we are doing this: I think it is not super obvious.

Reply:
does ```// purger doesn't purge prod clusters; this is how we know its a dev cluster by tagging "purge: true"``` suffice?

Reply:
I meant lines +76 to +79 specifically: explain the fact that we get a resource group to get existing tags so we remove them on `CreateOrUpdate` later.

Reply:
got it, adjusted comment